src/android/Device.java

```diff
@@ -18,19 +18,28 @@ Licensed to the Apache Software Foundation (ASF) under one
 */
 package org.apache.cordova.device;
 
-import java.util.TimeZone;
+import android.content.Context;
+import android.provider.Settings;
+import android.util.Log;
+
+import com.zebra.deviceidentifierswrapper.DIHelper;
+import com.zebra.deviceidentifierswrapper.IDIResultCallbacks;
 
-import org.apache.cordova.CordovaWebView;
 import org.apache.cordova.CallbackContext;
-import org.apache.cordova.CordovaPlugin;
 import org.apache.cordova.CordovaInterface;
+import org.apache.cordova.CordovaPlugin;
+import org.apache.cordova.CordovaWebView;
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
-import android.provider.Settings;
+import java.util.TimeZone;
 
 public class Device extends CordovaPlugin {
+
+    private CallbackContext _callbackContext;
+    private JSONObject _returnValue = new JSONObject();
```

Comment:
Do not keep these as state as this can cause issues with multiple threads. Instead, these should be passed around as parameters when required.